trainCycleGAN.py

```diff
@@ -172,10 +172,12 @@ def build_graph(image_shape, generator, discriminator):
         substitutions={real_Y_scaled.output: genG.output}
     )
     # genG( genF(X)) => Y     - fake_Y_loop
+
     genG_back = genG.clone(
         method='share',
-        substitutions={real_Y_scaled.output: genF.output}
```

Comment:
this change makes sense to me:
 genG = generator(real_X_scaled) usese real_X_scaled as input and we want to substitute it with genF(X).
Nice catch!